robot/web/web.go

```diff
@@ -324,7 +324,12 @@ func (svc *webService) StartModule(ctx context.Context) error {
 	streamInterceptors = append(streamInterceptors, opManager.StreamServerInterceptor)
 	// TODO(PRODUCT-343): Add session manager interceptors
 
-	svc.modServer = module.NewServer(unaryInterceptors, streamInterceptors)
+	opts := []googlegrpc.ServerOption{
+		googlegrpc.UnaryInterceptor(grpc_middleware.ChainUnaryServer(unaryInterceptors...)),
+		googlegrpc.StreamInterceptor(grpc_middleware.ChainStreamServer(streamInterceptors...)),
+		googlegrpc.UnknownServiceHandler(svc.foreignServiceHandler),
+	}
+	svc.modServer = module.NewServer(opts...)
```

Comment:
this is really the only actual change, the other being the change in method signature of module.NewServer()